pkg/database/memory/Handler.go

```diff
@@ -0,0 +1,34 @@
+package memory
+
+import (
+	"github.com/quan-to/chevron/pkg/models"
+	"github.com/quan-to/slog"
+	"sync"
```

Comment:
Same comment for [imports](https://github.com/golang/go/wiki/CodeReviewComments#imports).